package.json

```
@@ -3,7 +3,9 @@
     "compile": "hardhat compile",
     "test": "DRY_RUN=1 hardhat test",
     "benchmark": "DRY_RUN=1 hardhat test",
-    "benchmark:write": "hardhat test"
+    "benchmark:write": "hardhat test",
+    "lint:fix": "npx prettier . --write",
```

```suggestion
    "lint:fix": "prettier . --write",
```

Don't think `npx` is needed here.